helm/kubernetes-prometheus-chart/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+The Prometheus Operator has been installed. Check its status by running:
+  kubectl --namespace {{ .Release.Namespace }} get pods -l "release={{ .Release.Name }}"
+
+Visit https://github.com/giantswarm/prometheus-operator for instructions on how
```

Comment:
Not sure on this URL. It doesn't exist and I don't think this is a good name. See my main comment.